app/controllers/DefermentController.scala

```diff
@@ -78,6 +78,19 @@ class DefermentController @Inject()(identify: IdentifierAction,
     )
   }
 
+  private[controllers] def redirectToSplitPayment(userAnswers: UserAnswers): Result = {
+    userAnswers.get(UserTypePage).getOrElse("No user type") match {
+      case UserType.Importer => Redirect(controllers.routes.ImporterDanController.onLoad())
+      case _ =>
+        userAnswers.get(UnderpaymentTypePage).getOrElse("No underpayment type") match {
```

Comment:
there is a def called dutyType on the underpaymentType model that will do this for you.